### DISCUSSION TITLE
Can I use dxf-viewer to load DXF files from LibreDWG?

### DISCUSSION THREAD

**Discussion by billydotzhang** [vagran/dxf-viewer#112] [General]
Can I use [dxf-viewer](https://github.com/vagran/dxf-viewer) to load DXF files from [LibreDWG](https://github.com/LibreDWG/libredwg)?

**Comment by vagran** [comment]
Yes, I don't see any reason why it should not work. I also have a positive experience with using dxf-viewer for DXFs which are produced by `dwg2dxf` converter from LibreDWG package.

**Reply by billydotzhang** [reply]
This dwg file used `dwg2dxf`. bug something is not right.
[1.dwg.zip](https://github.com/user-attachments/files/16673346/1.dwg.zip)
![image](https://github.com/user-attachments/assets/a1c865c5-b862-472a-900d-92693c98a303)

**Reply by vagran** [reply]
I will check it.

**Reply by billydotzhang** [reply]
thx.

**Reply by billydotzhang** [reply]
We look forward to your early reply！ 🙏🏻

**Reply by vagran** [reply]
I tried to convert this file using `dwg2dxf` and then opening the resulting DXF file by AutoDesk Online viewer, it fails to open it. LibreCAD and QCAD managed to open, but also display it quite corrupted. This most probably indicates a problem in `dwg2dxf` implementation, not the `dxf-viewer`. In general, it is always better to compare DXF file display by some reference software and by the `dxf-viewer`, and then report visible inconsistency.
